### PR TITLE
Refactor refund

### LINKED ISSUE #6: Refactor VSDKRefund
Divide the Refund model class into TotalRefund and PartialRefund.

See: https://github.com/velocidi/velocidi-ios-objc-sdk/pull/3#discussion_r281253324

### LINKED ISSUE #13: Test if Examples compile
We should make a travis test to see if both Examples projects compile using the latest version of the SDK. This is to avoid editing methods and not editing them on the Examples, causing the Examples no to compile.

### LINKED ISSUE #19: Error printed on console when building documentation
When building the docs (`.scripts/build_docs.sh`), it prints an error on the console: `/Users/bmagalhaes/code/velocidi-ios-objc-sdk/VelocidiSDK/VelocidiSDK/VelocidiSDK.h:1:9: fatal error: 'UIKit/UIKit.h' file not found`

### PR DESCRIPTION
**Pull request by duartepinto**
This PR closes #6, closes #13 and closes #19

I've also added a CI test against iOS SDK 12.4

### CODE REVIEW

File: VelocidiSDK/VelocidiSDK/VelocidiSDK.modulemap [hunk 1]
```diff
@@ -1,5 +1,5 @@
 framework module VelocidiSDK {
-    umbrella header "VelocidiSDK.h"
+    umbrella header "VelocidiSDK-umbrella.h"
```
Comment (beatriz) [inline VelocidiSDK/VelocidiSDK/VelocidiSDK.modulemap:2]: Why this change? I'm curious as I was reading on umbrella headers and couldn't find an explanation 🙂

**Review by beatriz — APPROVED** [review]
LGTM, just left a small question.

File: VelocidiSDK/VelocidiSDK/VelocidiSDK.modulemap (hunk 1 above)
Comment (duartepinto) [reply VelocidiSDK/VelocidiSDK/VelocidiSDK.modulemap:2]: No specific reason, I just found it to be common to name this `<Framework's name>-umbrella.h` so I changed it.
